lua/lean/lean3.lua

```diff
@@ -49,8 +49,8 @@ local class_to_hlgroup = {
 
 -- mapping from lean3 events to standard div events
 local to_event = {
-  ["onMouseEnter"] = "mouse_enter";
-  ["onMouseLeave"] = "mouse_leave";
+  ["onMouseEnter"] = "cursor_entered";
```

Comment:
I think personally I very slightly prefer the older names, since below when you use `cursor_left`, in isolation it's easy to confuse that for something that's the opposite of `cursor_right`.